### DISCUSSION TITLE
Vacation Mode where is the 24h delay?

### DISCUSSION THREAD

**Discussion by KiefDelicious** [zachowj/node-red-contrib-home-assistant-websocket#465] [Q&A] [answered]
Hi, I really like your work. I'm implementing the vacation mode as described, but I can't find the delay of 24hours when someone isn't home in your workflow, so now the flow starts immediately when there is nobody home. Offcource I can add a delay node, but that doesn't seem very reliable. Do I miss something here?

(sidenote. I've added some extra fields to the Actionable Notification subflow, to add a picture(image) in the notify and a clickAction (what to do when you press the notification itself, that works amazing)

**Comment by zachowj** [comment, marked as answer]
I am assuming you're referring to this flow <https://zachowj.github.io/node-red-contrib-home-assistant-websocket/cookbook/vacation-mode.html#vacation-mode>.

It's in the events: state node _home/away_ in the `For` field.

![image](https://user-images.githubusercontent.com/37859597/131726100-7cefa201-85f0-4bb4-8ab6-efc8b00f43b3.png)

**Comment by KiefDelicious** [comment]
Ah, that explains it, I don't have that field, do you know how to add it? I can't find it. I have Node Red 2.0.5 and the plugin in HA say's version 10.0.0

![image](https://user-images.githubusercontent.com/8470212/131803253-0126171a-4d2d-44ab-b48a-4844ec482928.png)

**Reply by zachowj** [reply]
Sounds like you need to upgrade the home assistant nodes.

**Reply by zachowj** [reply]
Do you happen to have the ha node hard coded in your addon conf

**Reply by KiefDelicious** [reply]
Ah I see, I have updated the nodes now, and have that option., Thanks!
